pkg/apis/core/validation/validation.go

```diff
@@ -4967,6 +4967,13 @@ func ValidateResourceQuotaResourceName(value string, fldPath *field.Path) field.
 			return append(allErrs, field.Invalid(fldPath, value, isInvalidQuotaResource))
 		}
 	}
+
+	if r := strings.TrimPrefix(value, "count/"); r != value {
+		if len(strings.Split(r, ".")) == 1 && !helper.IsStandardQuotaResourceName(r) {
+			return append(allErrs, field.Invalid(fldPath, value, "use count/<resource>.<group> for resources from non-core groups"+
+				" or count/<resource> for resources from the core group"))
```

Comment:
1. the message is better to point out the problem directly, in this case, it is about "resource from non-core groups must use count/<resource>.<group>", not about the 2nd part "or count/<resource> for resources from the core group"
2. isInvalidQuotaResource used a string const, probably use a string const here too

Reply:
isInvalidQuotaResource message is generic, i think it's better to use another string isInvalidQuotaResourceName with the message something like:  "resource from non-core groups must use count/resource.group"

Reply:
i see the generic message isInvalidQuotaResource is still being used, could you change it to something more specific about the error?